web_action_conditionable/__manifest__.py

```diff
@@ -0,0 +1,12 @@
+{
+    "name": 'web_action_conditionable',
+    "version": "11.0.1.0.0",
+    "depends": [
+        'base',
+        'web',
+    ],
+    'data': ['views/view.xml'],
+    "author": "Cristian Salamea,Odoo Community Association (OCA)",
```

Comment:
'website': 'https://github.com/OCA/web'

Reply:
Thanks, I've made the change, though seems like Github lost track of it after the force-push.

Reply:
It's in the line below ;-)

Reply:
@StefanRijnhart yeah, the line is there, but when I click on the file name (above the piece of code), it sends me [here](https://github.com/OCA/web/pull/843/files/345c1e9d2b5a6948980b658e0e905579be65f901#diff-1c9df7f23577f3809216942b00713b51), which is the old commit. No worries, though, I don't think it'll affect the merge.

Reply:
Should be fine, indeed. It's the commit from when @simahawk reviewed the code.